app/services/create_newsletter_signup.rb

```diff
@@ -0,0 +1,46 @@
+require 'sendgrid-ruby'
+include SendGrid
+
+class CreateNewsletterSignup
+  include ActiveModel::Validations
```

Comment:
What do we need the validations for?

Reply:
I wondered that myself at one point when I was writing this. It was an unsure Junior Dev moment.